pkg/monitor/ovsdb.go

```diff
@@ -522,6 +517,7 @@ func (monitor *OVSDBMonitor) processOvsInterfaceUpdate(uuid string, rowupdate ov
 		InterfaceUUID: oldEndpoint.InterfaceUUID,
 		BridgeName:    oldEndpoint.BridgeName,
 		MacAddrStr:    oldEndpoint.MacAddrStr,
+		IPAddr:        oldEndpoint.IPAddr,
```

Comment:
Is this safe ? IPAddr is implement as []byte.

Reply:
done